WalletWasabi.Gui/Controls/NoparaPasswordBox.xaml

```diff
@@ -56,16 +56,31 @@
                 <Panel>
                   <TextBlock Name="watermark"
                              Opacity="0.5"
-                             Text="{TemplateBinding Watermark}"
-                             IsVisible="{TemplateBinding Text, Converter={x:Static StringConverters.IsNullOrEmpty}}" />
+                             Text="{TemplateBinding Watermark}">
+                    <TextBlock.IsVisible>
+                      <MultiBinding Converter="{x:Static BoolConverters.And}">
+                        <Binding RelativeSource="{RelativeSource TemplatedParent}"
+                                 Path = "WarningMessage"
+                                 Converter = "{x:Static StringConverters.IsNullOrEmpty}" />
+                        <Binding RelativeSource="{RelativeSource TemplatedParent}"
+                                 Path="Text"
+                                 Converter="{x:Static StringConverters.IsNullOrEmpty}" />
```

Comment:
What is the desired behaviour here? 
I think it should not show the watermark if there is a warning message. It should not depends on Text

Reply:
Yeah, it’s using an And to check for text and the warning message

Reply:
I messed this up with the floating watermark. Its OK.